lib/matplotlib/backends/_backend_tk.py

```diff
@@ -649,12 +649,63 @@ def _set_image_for_button(self, button):
         path_large = path_regular.with_name(
             path_regular.name.replace('.png', '_large.png'))
         size = button.winfo_pixels('18p')
+
+        # Nested functions because ToolbarTk calls  _Button.
+        def _get_color(color_name):
+            # `winfo_rgb` returns an (r, g, b) tuple in the range 0-65535
```

Comment:
```suggestion
            # `winfo_rgb` returns a 16-bit (r, g, b) tuple
```

Reply:
I like the explicit value better. It mirrors the constants used in the code below.

Reply:
I disagree; 8-bit and 16-bit are the more common terms for describing the range of colour values.

Reply:
I simply copied the wording from the [Tk documentation](https://www.tcl.tk/man/tcl/TkCmd/winfo.html#M30), okay with either personally.

Reply:
Prefer 65535, but 16-bit is also ok.

Reply:
I think either is clear, leaving as is to follow the tk docs.